### DISCUSSION TITLE
How to enable Spellchecker?

### DISCUSSION THREAD

**Discussion by zipswich** [GetPublii/Publii#1436] [Help] [answered]
Sorry for the rudimentary question. I did do a lot of search and read [the official document](https://getpublii.com/docs/website-settings.html), but I am still cannot figure out how to enable it. I got the impression that it is enabled by default, but I do not see any warning or correction after typing letter randomly. Could anyone offer a tip?

**Comment by internettips** [comment, marked as answer]
1. Start Publii.
2. On the left side column, choose Site settings (3rd from bottom).
3. In the main centre pane (Site settings), under Basic settings, there's a title called Enable spellchecker with a slider. Turn the slide on (click it). When turned on, the background colour of the slider changes to blue.
4. At the top right, choose Save Settings button.

**Reply by zipswich** [reply]
Thank you very much. It is working now. I don't know how I missed it because I checked that page.

**Reply by internettips** [reply]
No problem: easy done. You're welcome. :-)
